src/styles/blocks/card.scss

```diff
@@ -0,0 +1,123 @@
+// #region card
+
+.card {
+  width: 198px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__image {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+  }
+
+  &__text-container {
+    align-self: flex-start;
+    margin: 0 16px;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    padding-top: 40px;
+    color: $main-text-color;
```

Comment:
The variable `$main-text-color` is used here. Ensure that this variable is defined in your SCSS files before it is used, otherwise, it will cause a compilation error.